docs/source/reference/numpysupported.rst

```diff
@@ -264,6 +264,7 @@ The following top-level functions are supported:
 * :func:`numpy.concatenate`
 * :func:`numpy.convolve` (only the 2 first arguments)
 * :func:`numpy.copy` (only the first argument)
+* :func:`numpy.corrcoef` (only the 3 first arguments, requires NumPy >= 1.10 and SciPy >= 0.16)
```

Comment:
Perhaps this should have a note like `implementation behaves as per NumPy 1.11+` or similar ?

Reply:
Added a few words.